app/src/main/java/me/hufman/androidautoidrive/carapp/notifications/views/DetailsView.kt

```diff
@@ -49,6 +50,7 @@ class DetailsView(val state: RHMIState, val phoneAppResources: PhoneAppResources
 	fun initWidgets(listView: NotificationListView, inputState: RHMIState) {
 		state as RHMIState.ToolbarState
 		this.inputView = inputState
+		this.listView = listView
```

Comment:
It looks like this is saved only to get to `listView.state`, which is already available as `listState`

Reply:
agree, I missed that. (I guess t seems the temperature was a bit too high when I wrote that lines)

Reply:
removed